magento/pwa-studio
# Allow Magento HTTP Backend when configured

## Description

Small changes in order to allow http backend url - "**MAGENTO_BACKEND_URL**"

## Related Issue





Closes #2422

## Acceptance 



### Verification Stakeholders

### Specification


Changes to next packages:
- `upward-js`
- `pwa-buildpack`

### Verification Steps


1. Set http backend URL for **MAGENTO_BACKEND_URL**
2. Build project & run server
3. Verify that it works - catalog, images, place order.
4. Do the same for https.

## Checklist

* I have updated the documentation accordingly, if necessary.
[ ] I have added tests to cover my changes, if necessary.
